philosophy.html

```diff
@@ -0,0 +1,5 @@
+<h1>Our Notebook Philosophy</h1>
+<p>STScI's mission is to make cutting-edge astronomy accessible to everyone. Creating open-source Jupyter Notebooks and making them available on GitHub maximizes accessibility and community engagement. </p>
```

Comment:
I would add something about the purpose of the notebooks. Something along these lines...feel free to edit and make it better readable.

The developed Jupyter Notebooks serve multiple purposes:
- They are used to present science cases and workflows that can be useful to the community. Example use cases include foundational procedures (e.g., how to perform background subtraction) and scientific procedures focused on specific observing modes (e.g., spectral extraction and analysis with NIRSpec MOS).
- They demonstrate the existing functionality in core libraries and visualization tools.
- They expose the need for additional functionality and features that are useful for the community and should be incorporated into the core libraries and visualization tools.